wasm/naivevm/vm.go

```diff
@@ -176,7 +184,7 @@ func (vm *naiveVirtualMachine) exec(f *wasm.FunctionInstance) (errRet error) {
 	for vm.activeFrame != prevActive {
 		if buildoptions.IsDebugMode {
 			fmt.Printf("0x%x: op=%s (Label SP=%d, Operand SP=%d, Frame SP=%d) \n",
-				vm.activeFrame.pc, buildoptions.OptcodeStrs[vm.activeFrame.f.Body[vm.activeFrame.pc]],
+				vm.activeFrame.pc, buildoptions.OpcodeStrs[vm.activeFrame.f.Body[vm.activeFrame.pc]],
```

Comment:
ps do you think it is important to hide the string constants in debugmode? ack it will grow the code by an array of 256 (unless we trim it to fit)

Reply:
I think we don't need to hide the strings as it seems over optimization😄